test/SaveHandler/CacheTest.php

```diff
@@ -103,4 +103,17 @@ public function testReadWriteTwice()
 
         $this->assertEquals($this->testArray, unserialize($saveHandler->read($id)));
     }
+
+    public function testReadShouldAlwaysReturnString()
+    {
+        $cacheStorage = $this->prophesize('Zend\Cache\Storage\StorageInterface');
+        $cacheStorage->getItem('242')->willReturn(null);
+        $this->usedSaveHandlers[] = $saveHandler = new Cache($cacheStorage->reveal());
+
+        $id = '242';
```

Comment:
These are php basics @samsonasik :) Casting is not necessary. It is a string because of the quotes (`is_string('242') === true`). The contents of the string is not of importance here.